queries/julia/highlights.scm

```diff
@@ -107,19 +109,16 @@
 ;;; Keywords
 
 [
-  "abstract"
-  "const"
   "macro"
-  "primitive"
   "struct"
   "type"
-  "mutable"
   "where"
 ] @keyword
 
 "end" @keyword
 
-((identifier) @keyword (#any-of? @keyword "global" "local")) ; Grammar error
+((identifier) @keyword
+ (#any-of? @keyword "global" "local")) ; Grammar error
```

Comment:
isn't this `@storageclass`/`@type.classifier` (I assume global/local applies to variables vs actual types)

keyword is always fine in doubt :+1:

Reply:
I think they're akin to Lua's `local` and Python's `global` which are keywords.